### PR TITLE
Implement and test computation and display of overall statistics

### PR DESCRIPTION
**Pull request by yashpola**
## New Feature for #127 

Users can now view the following overall statistics using `view_overall_statistics`: 
- Number of applicants including breakdown by status
- Number of interviewers including breakdown by status
- Number of interviews

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ViewOverallStatisticsCommand.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        }
+
+        // instanceof handles nulls
+        return other instanceof ViewOverallStatisticsCommand;
+    }
+
+    @Override
+    public String toString() {
+        return new ToStringBuilder(this)
+                .toString();
```
Comment (zhikaiong2001) [inline src/main/java/seedu/address/logic/commands/ViewOverallStatisticsCommand.java:101]: Nice feature to have!

File: src/test/java/seedu/address/logic/commands/ViewOverallStatisticsCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        assertEquals(standardCommand, standardCommandCopy);
+
+        // null -> return false
+        assertNotEquals(standardCommand, null);
+
+        // different types -> return false
+        assertNotEquals(standardCommand, new ClearCommand());
+
+    }
+}
```
Comment (zhikaiong2001) [inline src/test/java/seedu/address/logic/commands/ViewOverallStatisticsCommandTest.java:62]: Good job in remembering to add tests for new command!

**Review by zhikaiong2001 — APPROVED** [review]
LGTM, great job!

**Review by headcube1 — APPROVED** [review]
LGTM

**Pull request merged**